titan_client/titan_stix/mappers/indicators.py

```diff
@@ -91,7 +91,10 @@ def map(self, source: dict) -> Bundle:
                 kill_chain_phases=[
                     KillChainPhase(kill_chain_name="mitre-attack", phase_name=tactics)
                 ],
-                custom_properties={"x_opencti_main_observable_type": mapping_config.opencti_type}
+                custom_properties={
+                    "x_opencti_main_observable_type": mapping_config.opencti_type,
+                    "x_opencti_score": self.settings.ioc_opencti_score,
```

Comment:
This property should only be added if `ioc_opencti_score` is present

Reply:
I pushed ioc_opencti_score is present check